oggm/core/models/flowline.py

```diff
@@ -300,6 +300,7 @@ def __init__(self, flowlines, mb_model=None, y0=0., fs=None, fd=None):
         """
 
         self.mb = mb_model
+        self.Aglen = Aglen
         self.fd = fd
```

Comment:
That should be fine, but we should clearly note in the code that this is for legacy backwards compatibility and make an effort to "switch" oggm to the modern notation.